docs/user/custom-domains.rst

```diff
@@ -2,125 +2,34 @@ Custom Domains
 ==============
 
 Custom domains allow you to serve your documentation from your own domain.
+For instance, this could be ``docs.example.com``.
 This is great for maintaining a consistent brand for your documentation and application.
 
-By default, your documentation is served from a Read the Docs :ref:`subdomain <hosting:subdomain support>` using the project's :term:`slug`:
+*By default*, your documentation is served from a Read the Docs :ref:`subdomain <hosting:subdomain support>` using the project's :term:`slug`:
 
 * ``<slug>.readthedocs.io`` for |org_brand|
 * ``<slug>.readthedocs-hosted.com`` for |com_brand|.
 
 For example if you import your project and it gets the :term:`slug` ``example-docs``, it will be served from ``https://example-docs.readthedocs.io``.
 
-.. contents:: Contents
-    :local:
+How does it work?
+-----------------
 
-Adding a custom domain
-----------------------
+To begin using a custom domain, you will need to create a new custom domain for your project and add a DNS record for your new custom domain.
+Once the DNS record has propagated, an SSL certificate will be automatically issued for your custom domain and your documentation will be configured to serve using the new domain and certificate.
 
-To setup your custom domain, follow these steps:
+In case you change your domain name, your documentation can have multiple secondary domains but only one canonical domain name.
+Additional domains or subdomains will redirect to the canonical domain.
 
-#. Go the :guilabel:`Admin` tab of your project.
-#. Click on :guilabel:`Domains`.
-#. Enter the domain where you want to serve the documentation from (e.g. ``docs.example.com``).
-#. Mark the :guilabel:`Canonical` option if you want use this domain
-   as your :doc:`canonical domain </canonical-urls>`.
-#. Click on :guilabel:`Add`.
-#. At the top of the next page you'll find the value of the DNS record that you need to point your domain to.
-   For |org_brand| this is ``readthedocs.io``, and for :doc:`/commercial/index`
-   the record is in the form of ``<hash>.domains.readthedocs.com``.
+What to consider
+----------------
```

Comment:
This is a new section, it's written as a reaction to a couple of unfortunate examples.

Reply:
I don't love this heading, it's very ambiguous, but I don't have a strong suggestion for improving it.

Reply:
What about something like "Considerations for choosing a custom domain" ?